src/ck-core/register.h

```diff
@@ -75,6 +75,7 @@ class EntryInfo {
       message type but store different data in it, like parameter
       marshalled messages.
```

Comment:
Maybe add an explanation of this variable to the comment here

Reply:
added in fixup